From 5aa0e68cf3074cf166c6ab563b33719fbe82f57c Mon Sep 17 00:00:00 2001
From: Hu Kejun <william.hu@rock-chips.com>
Date: Wed, 21 Nov 2018 18:20:11 +0800
Subject: [PATCH] arm64: dts: rockchip: px30: add grf for rkisp1

when connect to dvp camera, we need grf to config cif data width

Change-Id: I798d841149e1fd9f7c3fb58bd39d1706bf85ee5d
Signed-off-by: Hu Kejun <william.hu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/px30.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/px30.dtsi b/arch/arm64/boot/dts/rockchip/px30.dtsi
index 2b72de7962df..58e235229f54 100644
--- a/arch/arm64/boot/dts/rockchip/px30.dtsi
+++ b/arch/arm64/boot/dts/rockchip/px30.dtsi
@@ -1622,6 +1622,7 @@
 		devfreq = <&dmc>;
 		power-domains = <&power PX30_PD_VI>;
 		iommus = <&isp_mmu>;
+		rockchip,grf = <&grf>;
 		status = "disabled";
 	};
 
-- 
2.35.3

